lms/djangoapps/grades/models.py

```diff
@@ -529,7 +529,7 @@ class Meta(object):
     course_id = CourseKeyField(blank=False, max_length=255)
 
     # Information relating to the state of content when grade was calculated
-    course_edited_timestamp = models.DateTimeField(u'Last content edit timestamp', blank=False)
+    course_edited_timestamp = models.DateTimeField(u'Last content edit timestamp', blank=True, null=True)
```

Comment:
Nope, funnily enough course grades already allowed this field to be null.

https://github.com/edx/edx-platform/blob/e0085c9668b9e94235444a80fc263798acd188e1/lms/djangoapps/grades/new/course_grade.py#L35
https://github.com/edx/edx-platform/blob/e0085c9668b9e94235444a80fc263798acd188e1/lms/djangoapps/grades/models.py#L532